server/game/core/cost/CostAdjuster.ts

```diff
@@ -18,12 +18,12 @@ export enum CostAdjustType {
 interface ICostAdjusterPropertiesBase {
 
     /** The type of cards that can be reduced */
-    cardTypeFilter: CardTypeFilter;
+    cardTypeFilter?: CardTypeFilter;
 
     /** The type of cost adjustment */
     costAdjustType: CostAdjustType;
 
-    /** @deprecated (not yet tested) The number of cost reductions permitted */
```

Comment:
Did you end up testing this while you were developing? So long as you saw it work at least once I'm fine to remove the deprecated comment

Reply:
It was already tested as far as I can tell. Bendu's effect, for example, successfully discounts only the next card played.